src/components/PostList.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Post from './Post';
+
+// eslint-disable-next-line react/prop-types
+function PostList({ postsListArr }) {
+  return (
+    <div className="post__item">
+      {postsListArr.map(post => <Post post={post} key={post.id} />)}
```

Comment:
```suggestion
      {postsListArr.map(post => <Post postData={post} key={post.id} />)}
```
Три слова post подряд -  bad practice